packages/oscal-react-library/src/components/OSCALBackMatter.js

```diff
@@ -91,34 +65,82 @@ export default function OSCALBackMatter(props) {
       <OSCALBackMatterCard>
         <CardContent>
           <Grid container spacing={0}>
-            <Grid item xs={10}>
-              <TitleDisplay resource={resource} />
+            <Grid item xs={11}>
+              <TitleDisplay uuid={resource.uuid}>
+                <OSCALEditableTextField
+                  fieldName="title"
+                  canEdit={props.isEditable}
+                  editedField={
+                    props.isEditable
+                      ? [Object.keys(props.partialRestData)[0], "back-matter", "resources"]
+                      : null
+                  }
+                  editedValue={props.backMatter.resources}
+                  editedValueId={resource.uuid}
+                  onFieldSave={props.onFieldSave}
+                  partialRestData={
+                    props.isEditable
+                      ? {
+                          [Object.keys(props.partialRestData)[0]]: {
+                            uuid: props.partialRestData[Object.keys(props.partialRestData)[0]].uuid,
```

Comment:
Can you clarify why we need to pass the UUID in this body? it should not be necessary in the body of the PATCH?

Reply:
The `uuid` is oddly required for the rest call.